docs/notebooks/2_getting_started.ipynb

```diff
@@ -14,7 +14,7 @@
     "- `regrid_dataset` to regrid all data to a common grid.\n",
```

Comment:
<p>*two</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xscen/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
That one took me a while to see :). Fixed.